driver/web/docs/schemas/auth/ServiceAccount.yaml

```diff
@@ -1,11 +1,6 @@
 required:
-  - account_id
```

Comment:
Shouldn't `account_id` be `required`, `readyOnly`?

Reply:
No, because `account_id` is expected in the request body when creating a service account by copying an existing one.